todo_app/templates/index.html

```diff
@@ -2,17 +2,24 @@
 {% block title %}To-Do App{% endblock %}
 
 {% block content %}
-  <div class="jumbotron">
-    <h1 class="display-4">To-Do App</h1>
-    <p class="lead">Just another to-do app.</p>
-  </div>
+<div class="jumbotron">
+  <h1 class="display-4">To-Do App</h1>
+  <p class="lead">Just another to-do app.</p>
+</div>
+
 
-  <div class="row">
-    <div class="col-md-12">
-      <h2>Items</h2>
-      <ul class="list-group mb-4">
-        <!-- List the todo items here -->
-      </ul>
-    </div>
+<div class="row">
+  <div class="col-md-12">
+    <h2>Items</h2>
+    <ul class="list-group mb-4">
+      {% for item in items %}
+      <li>{{ item.title }}</li>
+      {% endfor %}
+    </ul>
+    <form action="/item" method="post">
+      <input type="text" id="title" name="title"><br>
```

Comment:
Really minor, but it'd be nice to make this `required` to provide some simple client-side validation
```suggestion
      <input type="text" id="title" name="title" required><br>
```

Side note, this sent me on a mild goose chase about whether `<br>` or `<br />` was better; sounds like it's very unimportant but there are [some mildly interesting differences in this SO discussion](https://stackoverflow.com/questions/1946426/html-5-is-it-br-br-or-br)!